wpilibj/src/main/java/edu/wpi/first/wpilibj/ADIS16448_IMU.java

```diff
@@ -670,6 +673,45 @@ public void reset() {
       m_integ_gyro_angle_x = 0.0;
       m_integ_gyro_angle_y = 0.0;
       m_integ_gyro_angle_z = 0.0;
+
+      if (m_simGyroAngleX != null) {
+        m_simGyroAngleX.set(0.0);
+      }
+      if (m_simGyroAngleY != null) {
+        m_simGyroAngleY.set(0.0);
+      }
+      if (m_simGyroAngleZ != null) {
+        m_simGyroAngleZ.set(0.0);
+      }
+
+      m_angleOffset = getGyroOrientation();
```

Comment:
Nit: Since we reset the angles to 0, we know that `getGyroOrientation()` will always be equivalent to `new Rotation3d()`.